File: K_Closest_Points_To_Origin.js
/**
 * @param {number[][]} points
 * @param {number} k
 * @return {number[][]}
 */
var kClosest = function(points, k) {
    let result =[]
    points.sort((a,b)=>{
         return (((a[0])**2 +(a[1])**2 ) >((b[0])**2 +(b[1])**2))?1:-1;
        //return (a[0]-a[1] > b[0]-b[1])?1:-1;
          }) 

          let i=0 ; 
          while(i<k){
             result.push(points[i])
             i++;
          }
    return result
 };

File: phase2/k-closest-points-to-origin.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        result = []

        points.sort(key=lambda a: a[0]**2 + a[1]**2)

        i = 0
        while i < k:
            result.append(points[i])
            i += 1

        return result